sdk/eventhub/event-hubs/samples/browserSample/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Do you know if we're supposed to add some sort of copyright header for these as well?

Reply:
That's a good question. @bterlson do we need copyright headers for our HTML files as well? If so, is putting the same comments as what we have in the JS files sufficient (using `<!-- -->` of course)

Reply:
Added the copyright!